src/mdx-pages/en/guild/linkwall.mdx

```diff
@@ -0,0 +1,49 @@
+---
+layout: "page"
+title: "Link Wall"
+lead: "All guild's chats and groups."
+---
+
+<meta name='robots' content='none'/>
```

Comment:
Setting the meta like this will put it into the body. 

The page template sets per page metadata with the Meta component. So passing the value through the frontmatter would be the way to go imo.